python/ray/serve/schema.py

```diff
@@ -63,6 +64,51 @@ def _route_prefix_format(cls, v):
     return v
 
 
+@PublicAPI(stability="alpha")
+class LoggingConfigSchema(BaseModel):
```

Comment:
Why is LoggingConfigSchema different from LogginConfig? Is it possible to unify them?

Reply:
I get rid of the config.py change and only keep the schema.py.